pymongo/write_concern.py

```diff
@@ -38,9 +38,9 @@ class WriteConcern:
           replicated to **two** secondaries). **w=0 disables acknowledgement
           of write operations and can not be used with other write concern
           options.**
-    :param wtimeout: (integer) Used in conjunction with `w`. Specify a value
-          in milliseconds to control how long to wait for write propagation
-          to complete. If replication does not complete in the given
+    :param wtimeout: (integer) **DEPRECATED** Used in conjunction with `w`.
```

Comment:
Let's also say MongoClient's wTimeoutMS option is deprecated too.

Reply:
Done